x-pack/plugins/security/server/authentication/authenticator.ts

```diff
@@ -421,14 +421,6 @@ export class Authenticator {
     return DeauthenticationResult.notHandled();
   }
 
-  /**
-   * Checks whether specified provider type is currently enabled.
-   * @param providerType Type of the provider (`basic`, `saml`, `pki` etc.).
-   */
-  isProviderTypeEnabled(providerType: string) {
```

Comment:
__note:__ we don't really need this method anymore, there is not much sense to keep it.